liquibase-standard/src/main/java/liquibase/util/StringUtil.java

```diff
@@ -580,7 +526,7 @@ public static boolean isAscii(String string) {
      * @return true if 7 bit-clean, false otherwise.
      */
     public static boolean isAscii(char ch) {
-        return ch < 128;
+        return StringUtils.isAsciiPrintable(Character.toString(ch));
```

Comment:
```suggestion
        // This cannot use Apache Commons here because Commons uses different (correct) criteria to determine if a character is ascii
        return ch < 128;
```